File: quadratic_equation.c
#include <stdio.h>
#include <math.h>

int main()
{
	float a, b, c, d, e, f, g, h;

	printf("Enter number A: ");
	scanf("%f", &a);
	printf("Enter number B: ");
	scanf("%f", &b);
	printf("Enter number C: ");
	scanf("%f", &c);
	d = 4*a*c;
	e = sqrt(fabs(b*b-d));
	f = 2*a;
	g = (-b + e)/f;
	h = (-b - e)/f;

	if (g == h){
		printf("Answer: %f", g);
	}
	else if (g == 0){
		printf("Answer: %f", h);
	}
	else if (h == 0){
		printf("Answer: %f", g);
	}
	else{
		printf("Answer: %f and %f\n", g, h);
	}

/*	printf("Number A: %f\n", a);
	printf("Number B: %f\n", b);
	printf("Number C: %f", c);*/

	return 0;
}

File: quadratic_equation.py
import math
a = input("Enter Number A:\n")
while not a:
	a = input("Enter Number A:\n")
b = input("Enter Number B:\n")
while not b:
	b = input("Enter Number B:\n")
c = input("Enter Number C:\n")
while not c:
	c = input("Enter Number C:\n")

a = float(a)
b = float(b)
c = float(c)

d = 4*a*c
e = math.sqrt(b*b-d)
f = 2*a
g = (-b + e)/f
h = (-b - e)/f

print([a, b, c])

if g == h:
	print("Answer: ", g)
elif g == 0:
	print(f"Answer: {h}")
elif h == 0:
	print(f"Answer: {g}")
else:
	print(f"Answer: {g} and {h}")